backend/java/main/DBRecordModel.java

```diff
@@ -0,0 +1,27 @@
+package main;
+public class DBRecordModel {
+    private final String nickname, city, zipCode, country;
```

Comment:
Have you checked what fields are necessary to get the data from Google Maps API ?

Reply:
![image](https://user-images.githubusercontent.com/36161561/220193254-a9ae1667-e635-43c2-b068-38304cb54b3b.png)

So we probably won't use Google Maps. We haven't decided what we're going to use yet, and the given data seems to make sense. We will correct it if necessary.